etc/packages.yml

```diff
@@ -1,6 +1,6 @@
 ---
 package:
-  name: "7.14"
+  name: "7.15"
```

Comment:
shoot, i think we have to split this in two PRs now.
one without this file that does a backport for the code changes.

and one that does backport the code changes.

another option: we mark this file exempt from backports:
https://github.com/elastic/detection-rules/blob/main/detection_rules/devtools.py#L142-L149

Reply:
I do like the idea of exempting it to keep us safe from any accidental backports of this file in the future. Additional idea: add a job to post a comment when the backport label added and an exempted change is included

Reply:
> Additional idea: add a job to post a comment when the backport label added and an exempted change is included

Yeah, I think this is a good long term improvement. While the PR is still open, comment with the files that are exempted from the backport. Then it's less hidden and magical. So basically run `--dry-run` and paste that in.

Until then, what would you prefer to do within this PR? Split in two or update the backport logic to exempt the file?

Or should I make a quick PR that we merge first that will exempt etc/package.yml?

Reply:
It would be a good real test, so if you have the cycles, making the PR and merging would be my preference

Reply:
PR to add the exemption
https://github.com/elastic/detection-rules/pull/1437